assets/src/scripts/web-services/flood-data.js

```diff
@@ -1,10 +1,11 @@
 import {get} from '../ajax';
 import config from '../config';
 
-
 export const FLOOD_EXTENTS_ENDPOINT = `${config.FIM_GIS_ENDPOINT}floodExtents/MapServer/`;
 export const FLOOD_BREACH_ENDPOINT = `${config.FIM_GIS_ENDPOINT}breach/MapServer/`;
 export const FLOOD_LEVEE_ENDPOINT = `${config.FIM_GIS_ENDPOINT}suppLyrs/MapServer/`;
+const waterwatchUrl = config.WATERWATCH_ENDPOINT;
```

Comment:
Please make this all uppercase constants as they are module level constants.

Reply:
Changed module level constants to be all upper case.